Большие перемены (часть 2)

Sasha20101989 opened the pull request:
---
:mortar_board: [Большие перемены (часть 2)](https://up.htmlacademy.ru/ecmascript/19/user/2033003/tasks/13)

Sasha20101989 committed:
поправил шаблон сортировки

Sasha20101989 committed:
перенес рендеринг сортировок в презентер из мэйна

Sasha20101989 committed:
реализовал обработчик

Sasha20101989 committed:
реализовал сортировку

Sasha20101989 committed:
отработал коменты

k0t-pirat commented on src/presenter/content-presenter.js:
```diff
@@ -39,10 +42,32 @@ export default class ContentPresenter {
+  #sortWaypoints(sortType){
+    switch(sortType){
+      case SortType.DAY:
+        this.#humanizedWaypoints.sort(sortWaypointByDate);
```
у тебя в сортировке используется массив this.#humanizedWaypoints, а в renderPoints - this.#waypointsByCheckedFilter. Из-за этого сортировка не имеет эффекта

k0t-pirat replied:
и вообще лучше все манипуляции с данными в модель убрать

Sasha20101989 replied:
Олег, я просто удалил с разметки проверку на checked

Sasha20101989 replied:
Эту часть с this.#waypointsByCheckedFilter я позже перепишу, нашел в будущих заданиях события по клику на фильтры и в ТЗ тоже есть инфа о том что фильтры меняются

k0t-pirat commented:
![image](https://user-images.githubusercontent.com/30899837/212274710-e02a9784-74c3-4bb7-b1fc-f1d20a1fc45d.png)
выбранный способ сортировки никак не обрабатывается в разметке

Sasha20101989 committed:
добавил отображение эффекта при смене сортировки

Sasha20101989 committed:
выполнил дополнительное задание

пока что сделал, так, в свободное время подумаю как собрать в один объект который буду передавать во вью

Sasha20101989 committed:
перенес в модель

Sasha20101989 committed:
исправил ошибку именования
